authority-portal-frontend/src/app/common/components/form-elements/organization-roles-tooltip/organization-roles-tooltip.component.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2024 sovity GmbH
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Contributors:
+ *      sovity GmbH - initial implementation
+ */
+import {Component} from '@angular/core';
+
+@Component({
+  selector: 'app-organization-roles-tooltip',
+  templateUrl: './organization-roles-tooltip.component.html',
+})
+export class OrganizationRolesTooltipComponent {
+  organizationRolesTooltip = `
+USER
+is the basic role that allows a person to open multiple participant related sections in portal but basically cannot change anything.
```

Comment:
is the basic role that allows a person to open multiple participant related sections in the portal.

Reply:
done

Reply:
"Basic role that allows access to multiple participant related sections in the portal"

Reply:
done